changelog.mdx

```diff
@@ -4,6 +4,144 @@ description: "All the nice stuff you asked for"
 icon: "sparkles"
 ---
 
+## 2024-08-09
+
+- `UI tweak` for first time users: Test result details of user's first test run are visible directly in project overview.
+
+<Frame caption="First time users project overview with test result detail, 08/2024">
+  <img
+    src="images/changelog/first-report.png"
+    alt="First time users project overview with test result detail"
+  />
+</Frame>
+
+## 2024-08-08
+
+- `Save & run`: New button to **run and save** an edited test case at once. You can click on the `run only` button if you want to validate the test first before saving it.
+
+<Frame caption="'save & run' the test to validate and save changes, 08/2024">
+  <img
+    src="images/editing/save-run.png"
+    alt="Save and run the test case"
+    width="300px"
+  />
+</Frame>
+
+<div class="mt-8" />
+
+<Frame caption="'run only' to validate the test before saving it, 08/2024">
+  <img
+    src="images/editing/save-run-only.png"
+    alt="Run only to validate the test before saving it"
+    width="300px"
+  />
+</Frame>
+
+## 2024-08-06
+
+- `AI suggesting more tests`: Similar to the AI discovery at set-up, we expanded the AI discovery feature to “spawn” individual test cases.
+
+<Frame caption="Have the AI agent suggest more test cases, 08/2024">
+  <img
+    src="images/setup/setup-11-suggest-more.png"
+    alt="have AI suggest more test cases"
+  />
+</Frame>
+
+- `Open status for new test cases`: In order to mark test cases coming out of **suggest more** as new (until you've opened them for the first time), we introduced a new column **open status** to properly persist the state.
```

Comment:
"... to properly persist state" !?!? why not just we introduce a new column.

Reply:
took it word by word from the release, sounded like we know what we were doing. I'll change it.